src/flask-api/net_predictor/WR_logistic_regression.py

```diff
@@ -0,0 +1,119 @@
+import pandas as pd
+from sklearn.linear_model import LogisticRegression
+from sklearn import metrics 
+import kenpom_creds as cred # A file (named "kenpom_creds.py") with proper credentials must be made in the "net-predictor" folder 
+from kenpompy.utils import login
+import kenpompy.summary as kpsum
+import kenpompy.misc as kpmisc
+import kenpompy.team as kpteam
+
+browser = login(cred.login_email, cred.login_password)
+training_season = 2020  # Choose which year's data the model is trained on
```

Comment:
Potentially we should make this into a function and use flags specified by the user to pass in these values:
https://docs.python.org/3/library/argparse.html